src/views/Main/Container.js

```diff
@@ -16,7 +16,7 @@ export class Container extends React.Component {
 
   constructor() {
     super();
-    Loom.configure({ baseUrl: 'localhost' });
+    Loom.configure({ baseUrl: 'api.loom.digital' });
```

Comment:
yeah I'm adding the environment service back. I just did this for right now if you needed an immediate fix/want to have the working ui